### PR TITLE
Support wagtail 4

### PR DESCRIPTION
**Pull request by nickmoreton**
Hi,

This updates the package for working with Wagtail v4 and Django v4.1

I also updated some other dependencies including Django, Isort and Sphinx and Jinja2

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
     url='https://github.com/takeflight/wagtail-schema.org',
 
     install_requires=[
-        'wagtail>=2.15,<4.0'
+        'wagtail>=2.15',
```
Comment (nickmoreton) [inline setup.py:25]: If you would prefer pinning the upper version I'm happy to change it.

File: setup.py [hunk 2]
```diff
@@ -46,6 +46,21 @@
         'Programming Language :: Python :: 3.9',
         'Programming Language :: Python :: 3.10',
         'Framework :: Django',
+        'Framework :: Django :: 3',
```
Comment (nickmoreton) [inline setup.py:49]: I thought this was worth doing.

File: tox.ini [hunk 3]
```diff
@@ -39,7 +44,7 @@ usedevelop = True
 deps =
 	{[base]deps}
 	isort
-commands = isort --recursive --check-only --diff wagtailschemaorg/ tests/
+commands = isort --check-only --diff wagtailschemaorg/ tests/
```
Comment (nickmoreton) [inline tox.ini:47]: --recursive is no longer required.
https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html

File: wagtailschemaorg/jinja2tags.py [hunk 4]
```diff
@@ -7,17 +7,24 @@
 else:
     from wagtail.core.models import Site
 
+from jinja2 import __version__ as jinja2_version
+
+if jinja2_version >= '3.0.0':
+    jinja2_context_function = jinja2.pass_context
```
Comment (nickmoreton) [inline wagtailschemaorg/jinja2tags.py:13]: Changed in v3+

**Review by nickmoreton — COMMENTED** [review]
I added some notes for the changes I have made.

**Review by katdom13 — APPROVED** [review]
LGTM

File: wagtailschemaorg/jinja2tags.py (hunk 4 above)
Comment (seb-b) [reply wagtailschemaorg/jinja2tags.py:13]: it's probably safe to default to the newer version, but nice for backwards compatibility 👍

**Pull request merged**